vrt/ci.js

```diff
@@ -0,0 +1,351 @@
+      pull_number: newSnapshotPullRequestNumber,
+      reviewers: [author.login],
+    });
+    _echo(`Requested review from \`${author.login}\` on new snapshot PR.`);
+  } catch (er) {
+    _echo(
+      `There was an error adding the original PR author as a reviewer for the new snapshot PR.`,
+    );
+    _echo(er);
+  }
+}
+
+async function removeSnapshotBranchFromGitHub() {
+  try {
+    await octokit.git.deleteRef({
+      owner: `uber`,
+      repo: `baseweb`,
+      ref: `heads/${SNAPSHOT_BRANCH}`,
+    });
+    _echo(`Removed the snapshot branch from GitHub`);
+  } catch (er) {
+    _echo(`There was an error removing the snapshot branch from GitHub.`);
+    _echo(er);
+  }
+}
+
+async function closeSnapshotPullRequest(snapshotPullRequestNumber) {
+  await notifySnapshotPullRequestOfClosure(snapshotPullRequestNumber);
+  try {
+    await octokit.pulls.update({
+      owner: `uber`,
+      repo: `baseweb`,
+      pull_number: snapshotPullRequestNumber,
+      state: 'closed',
+    });
+    _echo(`Closed the existing snapshot PR.`);
+  } catch (er) {
+    _echo(`There was an error closing the existing snapshot PR.`);
+    _echo(er);
+  }
+  await notifyOriginalPullRequestOfClosure(snapshotPullRequestNumber);
+}
+
+async function notifyOriginalPullRequestOfClosure(snapshotPullRequestNumber) {
+  try {
+    await octokit.issues.createComment({
+      owner: `uber`,
+      repo: `baseweb`,
+      issue_number: BUILDKITE_PULL_REQUEST,
+      body:
+        `Visual changes have been resolved. #${snapshotPullRequestNumber} has been closed. ` +
+        `A new snapshot branch will be created and a new PR will be opened if future commits on \`${BUILDKITE_BRANCH}\` trigger visual changes.`,
+    });
+    _echo(
+      `Posted a comment on original PR about closure of existing snapshot PR.`,
+    );
+  } catch (er) {
+    _echo(
+      `There was an error commenting on the original PR about snapshot resolution.`,
+    );
+    _echo(er);
+  }
+}
+
+async function notifySnapshotPullRequestOfClosure(snapshotPullRequestNumber) {
+  try {
+    await octokit.issues.createComment({
+      owner: `uber`,
+      repo: `baseweb`,
+      issue_number: snapshotPullRequestNumber,
+      body:
+        `Visual changes have been resolved. ` +
+        `This PR will be closed and \`${SNAPSHOT_BRANCH}\` will be deleted. ` +
+        `A new snapshot branch will be created and a new PR will be opened if future commits on \`${BUILDKITE_BRANCH}\` trigger visual changes.`,
+    });
+    _echo(
+      `Posted a comment on snapshot PR about visual resolution and closure.`,
+    );
+  } catch (er) {
+    _echo(
+      `There was an error commenting on the existing snapshot PR about snapshot resolution.`,
+    );
+    _echo(er);
+  }
+}
+
+async function addLabelsToNewPullRequest(newPullRequestNumber) {
+  try {
+    await octokit.issues.addLabels({
+      owner: `uber`,
+      repo: `baseweb`,
+      issue_number: newPullRequestNumber,
+      labels: [`bugfix`, `ci`],
```

Comment:
maybe we should create a new label?

Reply:
yeah good idea - maybe `snapshots` or `vrt`. although i am starting to not like `vrt` since it is not a commonly known abbreviation.